assignments/assignment01.rb

```diff
@@ -53,7 +54,14 @@ def titleize(s)
 # Write your own implementation of `reverse` called `my_reverse`
 # You may *not* use the built-in `reverse` method
 def my_reverse(s)
-  # your implementation here
+  string = s.split("")
+  loop = string.length
```

Comment:
loop is a reserved word in ruby, you should never use that as the name for a local variable.